ffig/generators/java.py

```diff
@@ -0,0 +1,51 @@
+# Generator module for Java.
+
+import ffig.generators
+import os
+
+
+def _render_template_to_file(filename, template_name, env, data):
+    template = env.get_template(template_name)
+    with open(filename, 'w') as output_file:
+        output_file.write(template.render(data))
+
+
+def generator(module_name, binding, api_classes, env, output_dir):
+    outputs = []
+
+    output_dir = os.path.join(output_dir, 'java', 'src', module_name)
+    if not os.path.exists(output_dir):
+        os.makedirs(output_dir)
```

Comment:
This _make-dir-if-not-there_ pattern is extremely common. Maybe we could / should have a utility module for such routines so they're always done consistently within FFIG?

Reply:
I have an impending PR where this would fit well.